.devcontainer/rstudio.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+echo "Waiting for rstudio-server to be ready.."
+
+# Get rstudio url and open it
+url=$(jq -r ".CODESPACE_NAME" /workspaces/.codespaces/shared/environment-variables.json)
+url_rstudio="https://$url-8787.app.github.dev"
```

Comment:
The url is not possible to reach without gitlab auth.

I will propose to keep it simple.

`$BROWSER "https://${CODESPACE_NAME}-8787.${GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN}"`

If you would like to check if rstudio is stareted we should query `http://localhost:8787`.